frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -23,7 +23,7 @@ export const validationSchema = yup.object().shape({
   }),
   application: yup.object().shape({
     name: yup.string(),
-    selectedKey: yup.string().required('Required'),
```

Comment:
Missed this change in this PR https://github.com/openshift/console/pull/2550#issuecomment-526789480